### PR TITLE
Assorted mispellings

### PR DESCRIPTION
**Pull request by korverdev**
Adds the following misspellings to the dictionary and their variants:
- optimiztaion
- annhiliate

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3802,6 +3802,11 @@ annhilates->annihilates
 annhilating->annihilating
 annhilation->annihilation
 annhilations->annihilations
+annhiliate->annihilate
+annhiliated->annihilated
+annhiliates->annihilates
+annhiliation->annihilation
+annhiliations->annihilations
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3809]:
```suggestion
annhiliations->annihilations
annhiliating->annihilating
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -38389,6 +38394,8 @@ optimizier->optimizer
 optimiziers->optimizers
 optimizies->optimizes
 optimiziing->optimizing
+optimiztaion->optimization
+optimiztaions->optimizations
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38398]: Perhaps add British spelling?
```
optimistaion->optimisation
optimistaions->optimisations
```

**Comment by korverdev** [comment]
@DimitriPapadopoulos Thanks for the suggestions! I've gone ahead and added those!

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @korverdev.

**Pull request merged**